packages/components/src/pages/Chat.vue

```diff
@@ -23,6 +23,10 @@ export default {
       type: String,
       default: DefaultApiURL,
     },
+    suggestions: {
```

Comment:
Will this have a default which is the current suggestions?

Reply:
It didn't. I added defaults in https://github.com/getappmap/appmap-js/pull/1512/commits/6e7bdf437399308ada6a8139b89db2b01ebdda92.